Kolskid standomix300 make meringue (resubmit)

dmbrasw2 opened the pull request:
(originally put this pull request in the universal one, instead of the copy I made for this branch. Removed old pull request, now resubmitting this one)

**Completed Work**

- Created new procedure: 'p-standomix300-make-meringue'
- Procedure's Goal: to make meringue cookies with the Stand'o'mix 300
- Laid out the basic sections and added first draft of steps
- Added images to repository and procedure

## User Scenario
"Chris has a small hobby baking business where he sells baked goods to friends, family, coworkers, and neighbors. He is a small-time baker and has only his home kitchen and appliances to make his baked goods. He wants to add meringue cookies to his offerings, but the process to make them by hand is too time consuming and tiring. He recently purchased a stand mixer to more efficiently make his current recipes, and has heard that a stand mixer can handle meringues efficiently and easily."

## Notable Review Requests:
- How is the layout so far, specifically with the ordered and unordered lists and alerts? I tried to make the alerts stand out more

- How are the steps, in terms of length and clarity?

- We're required to have images, so I tried to find ones that were constructive and supported the instructions. Are they? And how are the sizes?

Ademola-Adepoju commented on zitos/p-zitos-speedqueen-wash-laundry-p1.md:
```diff
@@ -1,46 +1,48 @@
-staging
+University Housing offers all residents living on-campus access to free laundry facilities. Please be mindful of other residents when doing your laundry in our common spaces.
+
+Residents must use their own detergents, dryer sheets, etc.
```
Hey @sezito2 good work!

My comment on this line "**_Residents must use their own detergents, dryer sheets, etc._**" is that: do you think it feels a little disconnected from the main paragraph? I get that it is supposed to be an Alerting move, yeah? Maybe  you want to make it part of the first step, like a caveat to when they're about to start washing.

Or maybe just make it clearer in it's current position by turning it into a "Note(s)" section.

Ademola-Adepoju reviewed (commented):
Hey @sezito2 good work!

My comment on this line "**_Residents must use their own detergents, dryer sheets, etc._**" is that: do you think it feels a little disconnected from the first paragraph? Maybe  you want to make it part of the first step, like a caveat to when they're about to start washing.

Or maybe just make it clearer in its current position by turning it into a "Note(s)" section. 

Also, so the readers don't get confused, do you want to list all the things they have to use that's theirs, instead of the etc.

justinwatsonn commented on zitos/p-zitos-speedqueen-wash-laundry-p1.md:
```diff
@@ -0,0 +1,48 @@
+1. Load your dirty clothes into the washing machine.
+   * **Optional**: separate clothes into whites and colors.
+  
+2. Add detergent to the washer.
```
I like your use of asterisk to indicate notes/considerations

justinwatsonn commented on zitos/p-zitos-speedqueen-wash-laundry-p1.md:
```diff
@@ -0,0 +1,48 @@
+## Drying your Clothes
+
+1. Load wet clothes into the dryer drum.
+   * **Alert**: Check the lint trap to ensure it is clean.
```
Could the alert possibly be a step, maybe the first?

dmbrasw2 committed:
Initial setup

Copied and renamed TOCOPY

dmbrasw2 committed:
Made branch 2

Renamed task 2 to make meringue and made a make meringue branch

dmbrasw2 committed:
Adding images

Added images to image repository, will put in file later

dmbrasw2 committed:
Fixed image filename

dmbrasw2 committed:
Meringue rough draft

Wrote a rough draft of the meringue procedure. Included a few images.
Added images to the meringue repository

dmbrasw2 committed:
Added a Pull Request file

Added the pull request file for Meringue to my repository

dmbrasw2 committed:
Moved images folder and PR text

Moved images from a folder I created to the provided assets/images/screenshots folder per instructions, and for better access across branches. Re-added images from the correct folder to the procedure.

+ Moved Pull request text from draft to the meringue-specifc PR

dmbrasw2 committed:
Fixed Images

Moved images to the correct, provided image folder that can be accessed by all branches, and then re-added the images from the new location to the procedure.

Ready for pull request review now

dmbrasw2 committed:
Grammar tweaks and bullet indentation; added a picture

Changed 'get rolling' to 'use it or lose it'

Indented bullets below steps

Added a picture of meringue being piped

gmsimon22 commented on kolskid/p-standomix300-make-meringue.md:
```diff
@@ -34,27 +34,30 @@ This is a reliable, tasty recipe that produces wonderful meringue cookies withou
-> **A note before things get rolling:** _Meringue will gradually lose its consistency as it sits around. Because of this, leftover meringue can’t be saved for a second batch of cookies, as it won’t be workable by the time the first batch is done. Plan to bake all of it at once._
+> **Ue it or Lose it:** _Meringue gradually loses its consistency as it sits around. Because of this, leftover meringue can’t be saved for a second batch of cookies, as it won’t be workable by the time the first batch is done. Plan to bake all of it at once._
 
 ### Make the Meringue
```
Could you change the subheadings to start with imperative action verbs? I think 'making the meringue' might sound nice and more active

alyssajohnsonunofficial replied:
I like Gabby's idea!

gmsimon22 commented on kolskid/p-standomix300-make-meringue.md:
```diff
@@ -34,27 +34,30 @@ This is a reliable, tasty recipe that produces wonderful meringue cookies withou
+> **Ue it or Lose it:** _Meringue gradually loses its consistency as it sits around. Because of this, leftover meringue can’t be saved for a second batch of cookies, as it won’t be workable by the time the first batch is done. Plan to bake all of it at once._
 
 ### Make the Meringue
 1. Wipe down the whisk and bowl with a dry cloth or paper towel, ensuring both are fully dry.
```
these are a lot of ordered list points at once. Is there a way that you could break up the content for readability. Maybe add a space between each.

gmsimon22 commented on kolskid/p-standomix300-make-meringue.md:
```diff
@@ -34,27 +34,30 @@ This is a reliable, tasty recipe that produces wonderful meringue cookies withou
+    - **Test for dissolved sugar:** Rub a small bit of meringue between your fingers. If it feels gritty, it needs to be mixed longer.
+    - **Test for stiff peaks:** Unhook the whisk and lift it straight out of the bowl. The meringue should come to a ‘peak’ that is stiff enough to stand upright when the whisk is turned over. If the peak flops over, it needs to be mixed longer.
+
 ![Stiff peaks staying upright even after the beater is turned over](<assets/images/screenshots/Meringue pics/Stiff peaks test.png>)
```
nice picture choices! Looks yummy

gmsimon22 commented on kolskid/p-standomix300-make-meringue.md:
```diff
@@ -34,27 +34,30 @@ This is a reliable, tasty recipe that produces wonderful meringue cookies withou
 ### Make the Cookies
 1. Transfer the meringue to the prepared piping bag.
-* Spooning it in with a flexible spatula or dough/bench scraper is easiest, especially with the bag already set up in a glass.
+    - Spooning it in with a flexible spatula or dough/bench scraper is easiest, especially with the bag already set up in a glass.
```
Maybe switch the word order to, "Try the easy method by spooning it..."
Could help for easy skimming

gmsimon22 commented on kolskid/p-standomix300-make-meringue.md:
```diff
@@ -34,27 +34,30 @@ This is a reliable, tasty recipe that produces wonderful meringue cookies withou
 2. Pipe equal(ish)-sized dollops onto the parchment paper about an inch apart. They need room for air circulation, but they will not spread.
-3. Place on the center rack of the preheated oven.
-4. Bake for 1 hour.
-5. Turn the oven off, but do not open the door. The residual heat will gently finish drying the cookies while the oven gradually cools down.
```
ooh nice tip! maybe the second sentence or warning could go in a sub point, to visually highlight it, or another way to highlight. Just a small suggestion though, it's also nice as is

gmsimon22 commented on kolskid/p-standomix300-make-meringue.md:
```diff
@@ -34,27 +34,30 @@ This is a reliable, tasty recipe that produces wonderful meringue cookies withou
-4. Bake for 1 hour.
-5. Turn the oven off, but do not open the door. The residual heat will gently finish drying the cookies while the oven gradually cools down.
-6. Leave the cookies in the off, closed oven for 1-2 hours until they cool completely.
-7. Enjoy!
```
Cute! I love it!

gmsimon22 reviewed (commented):
I've left some comments to this procedure, I hope it helps!

dmbrasw2 committed:
Peer review edits

Applied suggested edits from peer review

alyssajohnsonunofficial commented on kolskid/p-standomix300-make-meringue.md:
```diff
@@ -0,0 +1,62 @@
+# How to Make Meringue Cookies
+This procedure will guide you through the process of making tasty meringue cookies with the Stand’o’Mix 300. 
+
+This is a reliable, tasty recipe that produces wonderful meringue cookies without having to beat egg whites by hand until your arm falls off! Once bakers are more comfortable with the process, the cookies can easily be fancied up with food coloring, different flavors,and new cookie shapes.
```
These intros are so fun! Definitely indicates the intended audience

alyssajohnsonunofficial commented on kolskid/p-standomix300-make-meringue.md:
```diff
@@ -0,0 +1,62 @@
+  - Smooth-sided cookies: no piping tip needed.
+![Smooth meringue cookies](<assets/images/screenshots/Meringue pics/Smooth meringue.png>)
+  - Cookies with defined ruffles/ridges: star-shaped piping tip (any size) required.
+![alt text](<assets/images/screenshots/Meringue pics/Ruffle meringue.png>)
```
Love the placing of images here, and I think the unordered ingredient list looks well-organized and visually appealing

alyssajohnsonunofficial commented on kolskid/p-standomix300-make-meringue.md:
```diff
@@ -0,0 +1,62 @@
+  - Cookies with defined ruffles/ridges: star-shaped piping tip (any size) required.
+![alt text](<assets/images/screenshots/Meringue pics/Ruffle meringue.png>)
+
+### Ingredients
```
Also well-organized, and the list is nice and short, which makes it less intimidating for beginners

alyssajohnsonunofficial commented on kolskid/p-standomix300-make-meringue.md:
```diff
@@ -0,0 +1,62 @@
+### Prepare!
+1. Preheat the oven to 225F
+2. Separate the egg whites and yolks. The yolks can be used for a different recipe, or discarded.
+   > _**Tip:** Even a small amount of yolk can ruin a meringue. Separate the eggs over a ‘quarantine’ bowl, and then transfer the egg whites to a second ‘collection’ bowl. This way, if a yolk breaks, you won’t lose all the egg whites you’ve already separated._
```
Love this, really makes the tip/alert stand out

alyssajohnsonunofficial commented on kolskid/p-standomix300-make-meringue.md:
```diff
@@ -0,0 +1,62 @@
+5. Add in the vanilla extract.
+6. Beat until the mixture is glossy, has a thick, marshmallowy texture, and has increased in volume. It will take several minutes! 
+7. When the sugar is fully dissolved and the mixture comes to stiff peaks, it’s ready!
+    - **Test for dissolved sugar:** Rub a small bit of meringue between your fingers. If it feels gritty, it needs to be mixed longer.
```
More alerts that are visually distinct and helpful, and I love the wording of the steps here. I would suggest trying to adjust steps 4 and 7 to start with imperative verbs like the others, but I also totally understand if this isn't possible!

alyssajohnsonunofficial reviewed (commented):
These are all nicely fleshed out, well organized, and utilize effective/helpful images!

dmbrasw2 committed:
User scenario copy + Figure #s

Created copy of user scenario and renamed it to match file

Added figure #s and labels to images

dmbrasw2 committed:
Fraction formatting

Changed the formatting of some fractions. Not sure how to update 1/8 to match

dmbrasw2 committed:
User scenario fix

Copied user scenario wrong the first time. Deleted the miscopied file and made new ones

Filled out Meringue user scenario

dmbrasw2 committed:
More user scenario file problems

lingeringcode committed:
Feedback for Deanna

dmbrasw2 committed:
Applied changes per feedback

Numbered headings, fixed image nesting
